web/src/admin/applications/ApplicationForm.ts

```diff
@@ -107,6 +107,11 @@ export class ApplicationForm extends WithCapabilitiesConfig(ModelForm<Applicatio
                 },
             });
         }
+
```

Comment:
We should be doing both this and the change below in a more generic way for all objects that are identified by a user-configurable identifier cc @kensternberg-authentik